setup/Dockerfile

```
@@ -0,0 +1,54 @@
+FROM python:3.7-slim as base
+
+RUN apt-get update && apt-get install --no-install-recommends curl unzip git wget -y && apt-get clean && rm -rf /var/lib/apt/lists/*

```

I needed to install less to be able to make it work
```suggestion
RUN apt-get update && apt-get install --no-install-recommends curl unzip git wget less -y && apt-get clean && rm -rf /var/lib/apt/lists/*

```